Controller/PageAdminController.php

```diff
@@ -94,7 +94,9 @@ public function treeAction()
 
         $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
 
-        return $this->render('SonataPageBundle:PageAdmin:tree.html.twig', array(
+        $templates = $this->retrieveAdminTemplates();
+
+        return $this->render($templates['tree'], array(
```

Comment:
Because it's an array of parameters so the
```
$this->container->getParameter('sonata.page.templates.admin')['tree']
```
isn't available, that's why i made it in 2 step.

I can't
```
$this->admin->getTemplate('tree')
```
Because this parameters are related to SonataPageBundle and not SonataAdminBundle, the config node is templates_admin.

I don't know how to add this entries to SonataAdmin's `getTemplates()` collection from this bundle's file `SonataPageExtension.php`